core/src/main/resources/migration/tenant/4.0-2019-04-10-OC-10430.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+
+
+	<changeSet author="jkeremian" id="4.0-2019-04-10-OC-10430-1">
+		<addForeignKeyConstraint baseColumnNames="user_id"
+								 baseTableName="audit_log_event" constraintName="audit_log_event_user_by_id_fk"
+								 deferrable="false" initiallyDeferred="false"
+								 referencedColumnNames="user_id"
+								 referencedTableName="user_account" referencedTableSchemaName="public"/>
+	</changeSet>
+
+
+
+
+	<changeSet author="jkeremian" id="4.0-2019-04-10-OC-10430-2">
+		<addNotNullConstraint columnName="user_id"
```

Comment:
this is going to fail on the servers that already have null values, right?

Reply:
yes, it will fail for existing 0 and null values. We need to update those records in any schema that fails. thanks,Joe